<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelsForm</class>
 <widget class="QWidget" name="ModelsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QTableView" name="diffDbView">
     <property name="selectionMode">
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="m_diffFilter"/>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </item>
   <item row="2" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="m_refreshButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_savePictures">
       <property name="text">
        <string>Save pictures</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_saveOnePicture">
       <property name="text">
        <string>Save one picture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_removeSelectedDiffRow">
       <property name="text">
        <string>Remove selected rows</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_saveOneModel">
       <property name="text">
        <string>Save one model</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_saveDiffModels">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save models to stat db</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="3">
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
